src/views/LoginView.vue

```diff
@@ -203,19 +197,17 @@ export default {
       snackBarMessage: '',
       loading: false,
       showPass: false,
-      isFormValid:false,
-      usernameRules:[this.isAlphaNumeric],
-      passwordRules:[this.has8OrMoreCharacters],
+      isFormValid: false,
+      usernameRules: [this.isAlphaNumeric],
+      passwordRules: [this.has8OrMoreCharacters],
     };
   },
   computed: {
     isLoggingIn() {
       return this.$route.name === ROUTE_NAME_LOGIN;
     },
     formHeaderText() {
-      return this.isLoggingIn
-        ? this.t('login.formHeaderLogin')
-        : this.t('login.formHeaderSignup');
+      return this.isLoggingIn ? this.t('login.formHeaderLogin') : this.t('login.formHeaderSignup');
     },
     buttonText() {
```

Comment:
I think since now this is being used in two places a different name might be better.. maybe loginOrSignupText

Reply:
it just occurred to me that the second use of this is also a button... disregard...LOL